packages/fdc3-context/generated/context/ContextTypes.ts

```diff
@@ -644,6 +667,9 @@ export interface EntityData {
 /**
  * Free text to be used for a keyword search
  *
+ * @experimental classification of the instrument by type or category. SHOULD be one of the
+ * standardized values, although other string values are permitted.
+ *
```

Comment:
I feel like something may have gone wrong with the generation here? I'm not sure why this was added in so many places and seems to be associated with `interactionType`

Reply:
I think its been merged with `interactionType` by quick type as they are both reducing to string. 

However, I'm also really confused as to why we're getting multiple `/** */` comment blocks with no type code under them... The two things maybe related - perhaps its just not able/willing to generate a type representing just a string but is still outputting the comment??

We should check if this is happening in the initial generation or the post-processing @Roaders developed to apply to it - I suspect in the generation. Its already going on prior to your PR (as your comments are getting added to existing orphaned blocks) @hughtroeger, but I think is something we need to investigate!